src/averageAges.js

```diff
@@ -37,7 +41,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const arrWoman = (!withChildren)
+    ? people.filter((human) => human.sex === 'f')
+    : people.filter((mother) => mother.sex === 'f'
+      && (people.find((person) => mother.name === person.mother)));
```

Comment:
[checklist point 2](https://github.com/mate-academy/js_average-ages/blob/master/checklist.md)


Method `some` would be better use for this purpose instead of `find`, because this way as I understand you need only boolean, not entire person object.